index.html

```diff
@@ -1,49 +1,242 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
+  
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1">
-  <title>Week 1: HTML Tables</title><!-- Latest compiled and minified CSS -->
+  <title>National Homelessness</title>
+
   <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css">
+  <link href='https://fonts.googleapis.com/css?family=Open+Sans+Condensed:300,700' rel='stylesheet' type='text/css'>
+  
   <style>
-    .header {
-      text-align: center;
+    .section {padding: 1em 0;} 
+
+    .header {text-align: left;} 
+
+    #data-table td {width: 25%;}
+
+    .bar {height: 20px; background: #F67280;}
+
+    h1 {
+      color:#355C7D;
+      font-family: 'Open Sans Condensed', Helvetica, sans-serif;
+      font-size: 300%;
+    } 
+
+    p {
+      color:#355C7D;
+      font-family: 'Open Sans Condensed', Helvetica, sans-serif;
+      font-size: 90%;
     }
-  </style>
+
+    table {
+      border-collapse: collapse;
+      width: 100%;
+      white-space: nowrap; 
+      font-family: 'Open Sans Condensed', Helevetica, sans-serif;
+      font-weight: 500;
+      color: #F67280;
```

Comment:
Nothing against pink (😀), but pink over coral is very difficult to read, particular for users with limited visual acuity. You might check potential color combinations with this [color contrast checker](http://webaim.org/resources/contrastchecker/) to see whether they meet the a standard set of guidelines for minimum accessible contrast.